examples/chart/access/slack/templates/_helpers.tpl

```diff
@@ -60,3 +60,27 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{- define "slack.identitySecretName" -}}
+{{- if .Values.teleport.identitySecretName -}}
+{{- .Values.teleport.identitySecretName -}}
+{{- else if .Values.tbot.enabled -}}
+  {{- if .Values.tbot.defaultOutput.secretName }}
```

Comment:
I dropped the `secretName` value from the `tbot` chart. I figured it was simpler to stick with the chart based name.